common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java

```diff
@@ -87,14 +88,14 @@
         publisherFactory = new BiFunction<Class<? extends Event>, Integer, EventPublisher>() {
             
             @Override
-            public EventPublisher apply(Class<? extends Event> cls, Integer buffer) throws NacosException {
+            public EventPublisher apply(Class<? extends Event> cls, Integer buffer) throws NacosRuntimeException {
```

Comment:
Function signatures cannot throw exceptions

Reply:
I have already fixed it.